.storybook/config.js

```diff
@@ -18,12 +18,29 @@ function loadStories() {
   req.keys().forEach(filename => req(filename));
 }
 
+const fontPathMap = [
+  { prefix: 'F_REITH', path: '/fonts/Reith/' },
+  { prefix: 'F_NASSIM_ARABIC', path: '/fonts/Nassim/Arabic/' },
+  { prefix: 'F_NASSIM_PASHTO', path: '/fonts/Nassim/Pashto/' },
+  { prefix: 'F_NASSIM_PERSIAN', path: '/fonts/Nassim/Persian/' },
+  { prefix: 'F_NASSIM_URDU', path: '/fonts/Nassim/Urdu/' },
+  { prefix: 'F_ISKOOLA_POTA_BBC', path: '/fonts/IskoolaPota/' },
+  { prefix: 'F_LATHA', path: '/fonts/Latha/' },
+  { prefix: 'F_MALLANNA', path: '/fonts/Mallanna/' },
+  { prefix: 'F_NOTO_SANS_ETHIOPIC', path: '/fonts/NotoSansEthiopic/' },
+  { prefix: 'F_PADAUK', path: '/fonts/Padauk/' },
+  { prefix: 'F_SHONAR_BANGLA', path: '/fonts/ShonarBangla/' },
+];
+
 addDecorator(story => (
   /* eslint-disable react/jsx-filename-extension */
   <>
     <GlobalStyles
       fonts={Object.values(fontFaces).map(fontFace => {
-        return fontFace();
+        const fontMap =
+          fontPathMap.find(map => fontFace.name.includes(map.prefix)) ||
```

Comment:
Is this search needed?  

Would be more performant (and probably more readable) if it could be written like:
```
const fontPathMap = {
  F_REITH: '/fonts/Reith/',
  F_NASSIM_ARABIC: '/fonts/Nassim/Arabic',
  // etc
}
const fontMap = fontPathMap[fontFace.name] || fontPathMap.F_REITH;
```

Reply:
`fontFace.name` gets the full name of the font (Eg. `F_REITH_SANS_BOLD`) whereas we just want a partial match for the prefix (Eg. `F_REITH`) so unfortunately this search is still needed.